firefox-ios/Client/Frontend/Settings/AppearanceSettings/Generic views/GenericButtonCellView.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+
+struct GenericButtonCellView: View {
+    private let theme: Theme
+    private let title: String
+    private let onTap: () -> Void
+
+    private struct UX {
+        static let dividerHeight: CGFloat = 0.7
+        static var buttonPadding: CGFloat { 4 }
```

Comment:
Nit, but it feels strange to have both `=` and `{ }` here
```suggestion
        static let dividerHeight: CGFloat = 0.7
        static var buttonPadding: CGFloat = 4
```